### PR TITLE
8-PATCH/api/articles/article_id

### PR DESCRIPTION
**Pull request by holyseo**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -199,3 +199,67 @@ describe("POST/api/articles/:article_id/comments", () => {
...
+    const increaseVote = { inc_votes: "invalid" };
+    return request(app)
+      .patch("/api/articles/invalidId")
+      .send(increaseVote)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid request");
+      });
+  });
+  it("400 status - valid but non-existing article_id in the url", () => {
```
Comment (nc-poppy-m) [inline __tests__/app.test.js]: Is this a valid article_id ?

Comment (nc-poppy-m) [reply __tests__/app.test.js]: Also if something is valid but non-existent, what status code do we send back?

**Review by nc-poppy-m — CHANGES_REQUESTED** [review]
Generally very good testing, just a few changes to be made:
- See my feedback about non-existent vs invalid and what the difference is. We use 400 for one and 404 for the other.
- Ensure your endpoints.json tells the user (and you!) what a PATCH request to this endpoint will need to look like
- We need to get the updated article back, not just the votes property

Implement this feedback then resubmit and we'll take another look at it :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -93,6 +93,30 @@ describe("GET/api/articles", () => {
...
   });
+  it("status 400: responds with an error message when an article_id is invalid", () => {
+    return request(app)
+      .get("/api/articles/invalidId/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid request");
+      });
+  });
+  it("status 400: responds with an error message when an article does not exist", () => {
```
Comment (ghost) [inline __tests__/app.test.js:104]: Watch that test description

File: __tests__/app.test.js [hunk 3]
```diff
@@ -253,13 +277,14 @@ describe("PATCH/api/articles/:article_id", () => {
...
   });
   it("400 status - valid but non-existing article_id in the url", () => {
-    const increaseVote = { inc_votes: "invalid" };
+    const increaseVote = { inc_votes: 1 };
     return request(app)
-      .patch("/api/articles/invalidId")
+      .patch("/api/articles/99")
       .send(increaseVote)
-      .expect(400)
+      .expect(404)
```
Comment (ghost) [inline __tests__/app.test.js:284]: You've changed the code but not your test descriptions

File: __tests__/app.test.js [hunk 4]
```diff
@@ -253,13 +277,14 @@ describe("PATCH/api/articles/:article_id", () => {
...
       .then(({ body }) => {
-        expect(body.msg).toBe("invalid request");
+        console.log(body);
```
Comment (ghost) [inline __tests__/app.test.js:286]: Don't leave in console logs

**Review by ghost — APPROVED** [review]
Really make sure you're updating your test descriptions as well, not just the test.

**Pull request merged**